components/CreateAlertModal.tsx

```diff
@@ -258,6 +259,21 @@ const CreateAlertModal: FunctionComponent<CreateAlertModalProps> = ({
     actions.loadAlerts(mangoAccount.publicKey)
   }, [])
 
+  // Delete notifi Alerts that have fired
+  useEffect(() => {
+    const firedAlert = data?.alerts.find((item) => {
```

Comment:
Is it possible we find more than one fired alerts ?

Reply:
Yes, the useEffect hook would delete them serially, since calling `deleteAlert` will modify `data`